src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -282,6 +283,7 @@ export function extendSamplesWithCancerType(samples:Sample[], clinicalDataForSam
 }
 
 export const ONCOKB_ONCOGENIC_LOWERCASE = ["likely oncogenic", "predicted oncogenic", "oncogenic"];
+export const HOTSPOTS_TIMEOUT_MS = 5000;
```

Comment:
@jjgao @alisman @onursumer is this a good timeout? for oncoprint only, no timeout with the mutation table